packages/test/src/test-prometheus.ts

```diff
@@ -0,0 +1,47 @@
+import test from 'ava';
+import { v4 as uuid4 } from 'uuid';
+import fetch from 'node-fetch';
+import { WorkflowClient } from '@temporalio/client';
+import { DefaultLogger, NativeConnection, Runtime } from '@temporalio/worker';
+import * as activities from './activities';
+import { RUN_INTEGRATION_TESTS, Worker } from './helpers';
+import * as workflows from './workflows';
+
+if (RUN_INTEGRATION_TESTS) {
+  test.serial('Prometheus metrics work', async (t) => {
+    const logger = new DefaultLogger('DEBUG');
+    Runtime.install({
+      logger,
+      telemetryOptions: {
+        metrics: {
+          prometheus: {
+            bindAddress: '0.0.0.0:9090',
```

Comment:
Can we use a random port for this, and bind to 127.0.0.1 instead?

```
async function getPortFree() {
    return new Promise( res => {
        const srv = net.createServer();
        srv.listen(0, () => {
            const port = srv.address().port
            srv.close((err) => res(port))
        });
    })
}
```

Reply:
I knew I'd need to change that and forgot. Thanks for the reminder.